.mergify.yml

```diff
@@ -1,3 +1,13 @@
+shared:
+  - commit_message_template: &commit_message_template |
+      {{title}}
+      
+      {{body}}
```

Comment:
Is this full PR description? I think current PR template is noisy and is not suitable for commit message 🥵

Reply:
Yeah a little bit nosiy. May discuss here: 6791

Reply:
Adding comment in https://github.com/risingwavelabs/risingwave/issues/6791